docs/postgres.md

```diff
@@ -88,6 +88,36 @@ dropdb opencollective_test
 dropuser opencollective
 ```
 
+## Migrations
+
+When creating migrations and interacting with the database please follow the guidelines below;
+
+### Create a migration
+
+This will create a file in `migrations/` where you'll be able to put your migration and rollback procedures:
+
+```
+# The name of the migration should use kebab case
+
+npm run db:migration:create -- --name <name-of-your-migration>
+```
+
+**Note:** To create a migration, always use the above command, so that it aligns with the default [Sequelize](https://sequelize.org/) file naming conventions.
+
+### Run migrations
+
+This will run all the migrations in `migrations/`:
```

Comment:
It's not really all? Only the one that haven't run yet?

```suggestion
This will run all the pending migrations in `migrations/`:
```

Reply:
Ah yes, that makes sense. 👍🏽